src/main/java/startspringboot/demo/model/Comment.java

```diff
@@ -0,0 +1,145 @@
+package startspringboot.demo.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.ManyToOne;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Entity
+public class Comment {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Long commentId;
```

Comment:
```suggestion
    private Long internalId;
```
I may be a lit bit confusing with commentId